src/pages/learn/state-management/index.tsx

```diff
@@ -0,0 +1,200 @@
+  title,
+  description,
+  path,
+  "jumbotron": {
+    "background": images[0] {
+    	url
+  	},
+    image,
+    title,
+  	description,
+  	"name": subTitle
+  },
+  'theBigIdeas': resources[slug.current == 'the-big-ideas'][0]{
+    description,
+    title,
+    resources[] {
+    "name": subTitle,
+      ...
+    }
+ },
+ "recoilSection": resources[slug.current == 'state-management-with-recoil'][0]{
+  ...,
+  resources[]-> {
+    ...
+  }
+ },
+}`
+
+export async function getStaticProps() {
+  const data = await sanityClient.fetch(StateManagementQuery)
+
+  return {
+    props: {
+      data,
+    },
+  }
+}
+
+const RecoilSection = ({resource}: any) => {
+  const {title, description, resources, path} = resource
+
+  return (
+    <div className="mt-5 md:bg-gray-100 dark:bg-gray-700 rounded-lg py-10 p-5">
+      <Link href={path}>
+        <a
+          className="font-bold hover:text-blue-600 dark:hover:text-blue-300 transition ease-in-out"
+          onClick={() => {
+            track('clicked resource', {
+              resource: path,
+            })
+          }}
+        >
+          <h1 className="sm:text-2xl md:text-4xl text-xl font-extrabold leading-tighter text-center mb-4">
+            {title}
+          </h1>
+        </a>
+      </Link>
+
+      <Markdown
+        source={description}
+        allowDangerousHtml={true}
+        className="prose dark:prose-dark md:prose-lg md:dark:prose-lg-dark text-gray-900 dark:text-gray-100  mt-4 max-w-screen-md mx-auto text-center"
+      />
+      <div className="grid lg:grid-cols-3 grid-cols-1 gap-4 mt-10">
+        {resources.map((resource: any) => {
+          return (
+            <VerticalResourceCard
+              resource={resource}
+              className="mb-4 text-center"
+            />
+          )
+        })}
+      </div>
+    </div>
+  )
+}
+
+const BigIdeasSection = ({resource}: any) => {
+  const {title, description, resources} = resource
+
+  return (
+    <div className="">
+      <div className="grid lg:grid-cols-5 grid-cols-1 gap-10 bg-gray-100 dark:bg-gray-700 text-gray-700 dark:text-gray-50 rounded-lg shadow-sm px-5 sm:py-16 py-10 ">
+        <div className="col-span-2">
+          <h1 className="sm:text-2xl md:text-4xl text-xl font-extrabold leading-tighter text-center mb-4">
+            {title}
+          </h1>
+          <Markdown
+            source={description}
+            allowDangerousHtml={true}
+            className="prose dark:prose-dark md:prose-lg md:dark:prose-lg-dark text-gray-900 dark:text-gray-100  mt-4 max-w-screen-md"
+          />
+        </div>
+        <div className="col-span-3">
+          {resources.map((resource: any) => {
+            return <VideoCard resource={resource} className="mb-4" />
+          })}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+type JumbotronProps = {
+  resource: CardResource
+}
+
+const Jumbotron: FunctionComponent<JumbotronProps> = ({resource}) => {
```

Comment:
Is there a reason we define another Jumbotron here vs using the on in `/components`?

Reply:
Had a couple of issues passing the appropriate values. But that's definitely something to refactor the next time we work on this page.